app/assets/stylesheets/application.scss

```diff
@@ -1,6 +1,9 @@
 // govuk_admin_template
 @import "govuk_admin_template";
 
+// govuk_components
+@import "govuk_publishing_components/all_components";
```

Comment:
When using govuk_publishing_components it is conventional to use the components that you need rather than all as it reduces CSS size, could you switch to that? see: https://github.com/alphagov/govuk_publishing_components/blob/main/docs/install-and-use.md#4-include-the-assets